frontend/packages/yki/src/components/registration/steps/register/SubmitRegistrationDetails.tsx

```diff
@@ -21,12 +22,18 @@ const FillRegistrationDetails = () => {
     keyPrefix: 'yki.component.registration.registrationDetails',
   });
   const { isEmailRegistration } = useAppSelector(registrationSelector);
-  const { status: nationalitiesStatus } = useAppSelector(nationalitiesSelector);
+  const submitRegistrationStatus =
```

Comment:
Ei taida lopulta olla oikeastaan mitään väliä (suoritusajan puolesta), mutta kun en oo tutkiniut niin tuli mieleen että nämä vois yhdistääkin kun haetaan saman staten alta tietoja. Mut voi kai olla näinkin.

Reply:
Laitoin näin ihan vaan muuttujien nimeämisen kannalta. Mukavampi lukea kun ei tartte destrukturointia miettiä. Perffin kannalta lienee aika mitätön asia.